packages/react/src/primitives/Field/FieldClearButton.tsx

```diff
@@ -11,8 +11,8 @@ const FieldClearButtonPrimitive: Primitive<FieldClearButtonProps, 'button'> = (
   { ariaLabel = ariaLabelText, size, ...rest },
   ref
 ) => (
-  <FieldGroupIconButton ariaLabel={ariaLabel} ref={ref} {...rest}>
-    <IconClose size={size} />
+  <FieldGroupIconButton ariaLabel={ariaLabel} size={size} ref={ref} {...rest}>
```

Comment:
Great catch! This is definitely how it should have been done.

Reply:
Okay thanks! I remove the edits from the IconClose. If we don't add the size prop on IconClose in FealdClearButton, it should be fine.